scripts/select_appointments.sql

```diff
@@ -1 +1,16 @@
 -- retrieve appointment information
+SELECT d.given_name, a.appt_time
+FROM appointments a
+  INNER JOIN doctors d
+  ON d.id = a.doctor_id
+  INNER JOIN patients p
+  ON p.id = a.patient_id
+WHERE d.family_name = 'Hubbard';
+
+SELECT p.given_name, a.appt_time
+  FROM patients p
+  INNER JOIN appointments a
+  ON p.id = a.patient_id
+  INNER JOIN doctors d
+  ON d.id = a.doctor_id
+  WHERE p.family_name = 'Wilcox';
```

Comment:
Hmm, this looks correct but I didn't quite get the expected output from it. Here's the solution code but could just be that the information wasn't added correctly to the table.

```sql
-- retrieve appointment information
-- for Dr. Hubbard
SELECT p.family_name, a.date, a.time
  FROM patients p
  INNER JOIN appointments a
    ON a.patient_id = p.id
  INNER JOIN doctors d
    ON a.doctor_id = d.id
WHERE d.family_name = 'Hubbard';
-- for patient Wilcox
SELECT d.family_name, a.date, a.time
  FROM doctors d
  INNER JOIN appointments a
    ON a.doctor_id = d.id
  INNER JOIN patients p
    ON a.patient_id = p.id
WHERE p.family_name = 'Wilcox';
```